src/couch_replicator/src/couch_replicator_docs.erl

```diff
@@ -695,7 +695,8 @@ strip_credentials(Url) when is_binary(Url) ->
         "http\\1://\\2",
         [{return, binary}]);
 strip_credentials({Props}) ->
-    {lists:keydelete(<<"oauth">>, 1, Props)}.
+    Props0 = lists:keydelete(<<"oauth">>, 1, Props),
```

Comment:
Let's remove the bugzid bit from the commit as well.

Reply:
Yes, removed.